packages-backend/eslint-config-node/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 commercetools GmbH
```

Comment:
```suggestion
Copyright (c) 2022 commercetools GmbH
```

Reply:
Yeah I'll do this as a follow up.

PS: I'm wondering if we can automate this with a github action or something that runs periodically once a year.